feature/gribi/ate_tests/ack_in_presence_other_routes/route_ack_test.go

```diff
@@ -197,16 +197,14 @@ func configureNetworkInstance(t *testing.T) {
 }
 
 // configStaticRoute configures a static route.
-func configStaticRoute(t *testing.T, dut *ondatra.DUTDevice, prefix string, nexthop string) *telemetry.NetworkInstance_Protocol {
-	ni1 := dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance).Get(t)
-	static := ni1.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, *deviations.StaticProtocolName)
+func configStaticRoute(t *testing.T, dut *ondatra.DUTDevice, prefix string, nexthop string) {
+	nInst1 := dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance).Get(t)
+	static := nInst1.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "DEFAULT")
```

Comment:
Keep the *deviations.StaticProtocolName change that was picked up in the rebase.

Reply:
Using *deviations.StaticProtocolName now in both ate and otg tests